recipe/meta.yaml

```diff
@@ -9,13 +9,14 @@ source:
   sha256: c2de3a60e9e7d07be26b7f2b00ca0309c207e06c100f9cc2a94931fc75a478fc
 
 build:
-  number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 2
```

Comment:
number 1 ?

Reply:
actually I think you messed up with branches, you should branch out from master not main 0.16 and your build number should be +1 of this one https://github.com/AnacondaRecipes/docutils-feedstock/blob/35c78f430afb2e780a84f9ef9929ad1de455b690/recipe/meta.yaml#L13

Reply:
speaking with Jack actually it is good  as he need old version of this